locale/dbtranslate.php

```diff
@@ -124,7 +124,7 @@
 echo __('Email maintenance alerts for users with view permissions to effected Displays.');
 echo __('Set the level of logging the CMS should record. In production systems "error" is recommended.');
 echo __('Which Calendar Type should the CMS use?');
-
+echo __('Default Delete Old Version of media');
```

Comment:
Can this be:

Default for "Delete old version of Media" check box. Shown when Editing Library Media.